### PR TITLE
feat: "/learn" site changes

### PR DESCRIPTION
**Pull request by mohandast52**
- Preview [here](https://olas-website-git-mohan-learn-page-updates-autonolas.vercel.app/learn)
- Few improvements 
- Deleted unused images

https://github.com/valory-xyz/olas-website/assets/22061815/5226f93c-1079-43d7-b6eb-fbef9e664404

### CODE REVIEW

File: components/LearnPage/SovereignAgents.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+export const SovereignAgents = () => (
+  <SectionWrapper
+    customClasses="px-4 py-12 border-y lg:px-24 lg:p-16 border-t"
+    id="sovereign-agents"
+  >
+    <div className={`${SCREEN_WIDTH_LG} gap-6 justify-center`}>
+      <h2
+        className={`${SUB_HEADER_CLASS} mb-12 self-center max-w-[740px] md:text-center`}
+      >
+        What different types of system can be build on Olas?
```
Comment (truemiller) [inline components/LearnPage/SovereignAgents.jsx:19]:
```suggestion
        What different types of system can be built on Olas?
```

File: components/Content/Article.jsx [hunk 2]
```diff
@@ -1,41 +1,72 @@
...
 
-  const { title, datePublished, slug } = article.attributes;
-  const image = article?.attributes?.headerImage?.data[0]?.attributes?.formats?.large;
+  const image = useMemo(() => {
+    const imageData = article?.attributes?.headerImage?.data;
+    const data = isArray(imageData) ? imageData?.[0] : imageData;
+
+    if (!data) return null;
+
+    return data?.attributes?.formats?.large;
```
Comment (Tanya-atatakai) [inline components/Content/Article.jsx:22]: wonder won't it be easy just to return `article?.attributes?.headerImage?.data[0]?.attributes?.formats?.large;`
wdyt

**Review by oaksprout — DISMISSED** [review]

**Review by truemiller — APPROVED** [review]

File: components/LearnPage/HowDoDecentralizedAgentsWork.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  'Services get data from any data source',
+  'They can do complex stuff, even machine learning',
+  'Service agents robustly come to consensus about what action to take',
+  'Services take action on any chain or call APIs',
+];
+
+export const HowDoDecentralizedAgentsWork = () => (
+  <SectionWrapper customClasses="lg:p-24 px-4 !py-0">
+    <div className={`${SCREEN_WIDTH_LG}`}>
+      <div className={`${SUB_HEADER_MEDIUM_CLASS} mb-8`}>
```
Comment (Tanya-atatakai) [inline components/LearnPage/HowDoDecentralizedAgentsWork.jsx:23]: can be header or span tag?

**Review by truemiller — APPROVED** [review]

File: components/LearnPage/HowDoDecentralizedAgentsWork.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        How do decentralized agents work?
+      </div>
+    </div>
+
+    <div className={`${SCREEN_WIDTH_XL}`}>
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+        <div className="flex-1">
+          <div className="img-container">
```
Comment (Tanya-atatakai) [inline components/LearnPage/HowDoDecentralizedAgentsWork.jsx:31]: maybe can remove this extra tag? and move `img-container` to it's parent with `flex-1`

**Review by Tanya-atatakai — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**